test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -30,6 +30,7 @@ const ignoredMocks = [
 
   // OKTA-535435 "Sign me out of all other devices." (enroll-authenticator.okta_password.credentials.revokeSessions) has no entry in login.properties
   'authenticator-enroll-password.json',
+  'authenticator-expired-custom-password.json' // seems to be flaky
```

Comment:
could you add a little more info on what flakiness you saw? Maybe just a short hint so whoever looks at this has more context... or you can file a jira too with all the details

Reply:
I am opening up a ticket with the details

Reply:
@LazarVuckovic-okta sync offline and will file a JIRA to follow up the flakiness test.